### PR TITLE
[DO NOT MERGE] Release more wheels

### PR DESCRIPTION
**Pull request by icemac**
Trying to release wheels for manylinux and Mac OS.

The code was taken from `zope.interface`, thus it might be broken there, too.

### CODE REVIEW

**Comment by mgedmin** [comment]
FWIW you do not need to create a pull request to get Travis/Appveyor to do a build.  It's enough to push a branch.

**Comment by mgedmin** [comment]
"exec: \"/io/.manylinux-install.sh\": permission denied" -- is the file marked executable?  I didn't pay attention to file permission bits in the original PR.

File: .travis.yml [hunk 1]
```diff
@@ -7,9 +7,65 @@ python:
...
-    - python setup.py -q test -q
+    - coverage run setup.py test -q
 notifications:
     email: false
+after_success:
+    - coveralls
+    - echo [distutils]                                  > ~/.pypirc
+    - echo index-servers = pypi                        >> ~/.pypirc
+    - echo [pypi]                                      >> ~/.pypirc
+    - echo repository=https://pypi.python.org/pypi     >> ~/.pypirc
```
Comment (mgedmin) [inline .travis.yml]:
```
Note: you are uploading to the old upload URL. It's recommended to use the new URL "https://upload.pypi.org/legacy/" or to leave the URL unspecified and allow twine to choose.
Uploading zodbpickle-1.0-cp27-cp27m-macosx_10_6_intel.whl
HTTPError: 410 Client Error: Gone (This API has been deprecated and removed from legacy PyPI in favor of using the APIs available in the new PyPI.org implementation of PyPI (located at https://pypi.org/). For more information about migrating your use of this API to PyPI.org, please see https://packaging.python.org/guides/migrating-to-pypi-org/#uploading. For more information about the sunsetting of this API, please see https://mail.python.org/pipermail/distutils-sig/2017-June/030766.html) for url: https://pypi.python.org/pypi
```

I think Twine should know the right upload URL to use, so how about let's remove the `echo repository=...` override from .pypirc here (and in master)?

Comment (mgedmin) [reply .travis.yml]: And, yes, we ought to do that from every other zopefoundation project that uploads wheels from Travis:

```
mg@platonas: ~/src/zopefoundation $ grep -l twine */.travis.yml
BTrees/.travis.yml
persistent/.travis.yml
zope.index/.travis.yml
zope.interface/.travis.yml
```

**Comment by mgedmin** [comment]
> FWIW you do not need to create a pull request to get Travis/Appveyor to do a build. It's enough to push a branch.

OTOH if you'd pushed to a branch without creating a PR, I wouldn't have seen the failures.

(Maybe that would be a plus, I can be overly chatty sometimes.)

> "exec: "/io/.manylinux-install.sh": permission denied" -- is the file marked executable? I didn't pay attention to file permission bits in the original PR.

I don't think GitHub even shows the file permission bits!

**Comment by icemac** [comment]
@mgedmin Thank you for looking into this PR. I did the fixes you suggested. Let's see what happens.

**Comment by mgedmin** [comment]
I see a lot of wheels at https://pypi.python.org/pypi/zodbpickle

:1st_place_medal:

File: .travis.yml (hunk 1 above)
Comment (icemac) [reply .travis.yml]: @mgedmin Thank you for digging this up. I remove the line from each of those config files.

**Pull request closed without merging**